app/views/providers/allocations/edit_repeat_request.html.erb

```diff
@@ -0,0 +1,65 @@
+
+<% page_title = "Do you want to request PE for this organisation?"%>
+<%= content_for :page_title, page_title%>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(provider_recruitment_cycle_allocations_path) %>
+<% end %>
+
+<% legend = capture do %>
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+        <h1 class="govuk-fieldset__heading">
+            <span class="govuk-caption-xl"><%= @training_provider.provider_name %></span>
+            <%= page_title %>
+        </h1>
+    </legend>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+  <%= form_with url: provider_recruitment_cycle_allocation_path(
```

Comment:
GOVUK form builder candidate?

Reply:
@aliuk2012 ?

Reply:
Sorry only just seen this. We should use the form builder